ui/helpers/utils/snaps.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Check if the given value is a valid snap ID.
+ *
+ * NOTE: This function is a duplicate oF a yet to be released version in @metamask/snaps-utils.
+ *
+ * @param value - The value to check.
+ * @returns `true` if the value is a valid snap ID, and `false` otherwise.
+ */
+export function isSnapId(value: unknown): value is string {
```

Comment:
This function is available in `snaps-utils` now

Reply:
I just converted the file to typescript, it's used in too much places to make this change in this PR

Reply:
Fair, do you mind creating a ticket so we don't forget?

Reply:
Will do when I can load the new issue page

Reply:
https://github.com/MetaMask/metamask-extension/issues/29280